diff --git a/MODULE.bazel b/MODULE.bazel
index 7c85d01..256bed2 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -10,5 +10,6 @@ bazel_dep(name = "platforms", version = "1.0.0")
 bazel_dep(name = "bazel_skylib", version = "1.8.1")
 bazel_dep(name = "bazel_lib", version = "3.1.0")
 bazel_dep(name = "jq.bzl", version = "0.4.0")
+bazel_dep(name = "rules_shell", version = "0.3.0")
 
 deb_archive = use_repo_rule("//internal:deb.bzl", "deb_archive")
diff --git a/apt/private/deb_download.bzl b/apt/private/deb_download.bzl
index a9a4e9b..cf3b103 100644
--- a/apt/private/deb_download.bzl
+++ b/apt/private/deb_download.bzl
@@ -52,6 +52,8 @@ def _get_input_data_with_sources(rctx):
     return input_data
 
 _INDEX_BUILD_TMPL = """
+load("@rules_shell//shell:sh_binary.bzl", "sh_binary")
+
 filegroup(
     name = "lockfile",
     srcs = ["lock.json"],
